README.md

```diff
@@ -26,3 +22,24 @@ https://github.com/comocheng/kineticmodelssite/wiki/Making-a-Local-Version-of-th
 ## Importing:
 To import RMG models from the importer project
 - python scripts/import_RMG_models.py path/to/your/RMG-models/
+
+## Contributing:
+
+### Dev Environment
+To start contributing, clone the project and setup a conda environment.
+Then activate the environment and `pip` install the dev requirements:
+
+```pip install -r requirements-dev.txt```
+
+We use [black](https://github.com/psf/black) to format our codebase.
+To format the codebase, `cd` to the project root and run:
+
+```black .```
+
+If you add a `.py` file that you don't want formatted,
+add it to the `tool.black` section of the `pyproject.toml`.
+
+### Version Control Strategy
+1. Create a feature branch from `master`
+2. When finished, rebase the feature branch to the tip of `master`
+3. Open a PR and perform a Squash & Merge once approved
```

Comment:
Do we really want to Squash? Ideally the individual commits and messages are informative...

Reply:
I figure that it is just as easy to read the PR instead of keeping all the commits in the history. I should probably make the PR titles a bit more informative though and add better descriptions. That's just my opinion though.